### ISSUE TITLE
Obtain inventory for reporting

### ISSUE THREAD

**Issue by chargio** [SaltCellar/SaltCellar#17]
As a user, I want to create a report for a defined period of time.
I.e. I want to know all the objects for the account X in AWS that will be used for charging.
The user will define the time period (i.e. 2017/01/01 - 00:00:00 to 2017/01/31 - 23:59:59), and the user or object associated. It will get all relevant objects that were part of the account during that period, including those created, already existing, and destroyed during the period.

## Acceptance Criteria

- [ ] Verify that all objects are identified
- [ ] Verify that the objects include creating and destruction times (i.e. an object created 2 days after the start of the period and destroyed 3 hours before the end will have both dates and time properly identified)
- [ ] The time is identified, at least, with the same definition of the provider (i.e. AWS or GCE per second, other per hour)

## Assumptions and Questions

- There is some service that provides the inventory, accessible through an API.

**Comment by chargio** [comment]
Yes. I expect this to be split in actionable items during our sprint preparation meetings, and then be acted upon.

**Comment by blentz** [comment]
Here's an old script I wrote to scan for EC2 inventory: https://github.com/blentz/cloud-costs/blob/master/budget/scripts/aws_inventory_update.py

This was built on Pyramid and SQLAlchemy. It may be useful only as a guide and may not be very reusable.
